region/norfair/east.json

```diff
@@ -1869,25 +1903,101 @@
           "from": 4,
           "to": [
             {
-              "id": 1,
-              "requires": null,
+              "id": 3,
+              "requires": [
+                {"or": [
+                  "heatProof",
+                  "canHeatRun"
+                ]},
+                {"cost": [
+                  {"heatFrames": 220}
+                ]}
+              ],
               "unlock": null
             },
             {
-              "id": 3,
-              "requires": null,
+              "id": 6,
+              "requires": [
+                {"or": [
+                  "heatProof",
+                  "canHeatRun"
+                ]},
+                {"cost": [
+                  {"heatFrames": 200}
+                ]}
+              ],
               "unlock": null
             }
           ]
         },
         {
           "from": 5,
+          "to": [
+            {
+              "id": 6,
+              "requires": [
+                "Morph",
+                {"or": [
+                  "heatProof",
+                  "canHeatRun"
+                ]},
+                {"or": [
+                  {"and": [
+                    "ScrewAttack",
+                    {"cost": [
+                      {"heatFrames": 780}
+                    ]}
+                  ]},
+                  {"and": [
+                    "PowerBomb",
+                    {"cost": [
+                      {"heatFrames": 850},
+                      [
```

Comment:
It makes sense to have an array for ammo, but it's "ammo" itself that has to be an array, rather than be in an array.